app/views/AuthenticationWebView.js

```diff
@@ -110,7 +111,10 @@ class AuthenticationWebView extends React.PureComponent {
 			<>
 				<StatusBar theme={theme} />
 				<WebView
-					source={{ uri }}
+					source={{
+						uri,
+						headers: RocketChatSettings.customHeaders
+					}}
```

Comment:
It'll open a third-party url, can you check if it's needed?

Reply:
Will look into this